### ISSUE TITLE
"Reverse" Split (work in progress stays in original PBI)?

### ISSUE THREAD

**Issue by vitpace** [snuziale/vsts-extension-split-work#33]
Hi there,

Would it be possible to support a "reverse" split (e.g. _move_ the tasks that are done to a **new PBI** and **keep the original PBI** with the work that is not done yet)?

The reason behind it: related work done outside of Azure DevOps (like linking commits to the story etc) could then keep using the original ID.

There was also a somewhat related request under [Q&A below ](https://marketplace.visualstudio.com/items?itemName=blueprint.vsts-extension-split-work&ssr=false#qna)about the commits - in my interpretation of the split, all commits would be either linked to both PBIs or would stay with the original one (which will be used to continue work).

PS if this one is supported, i suspect #8 will automatically be addressed as original PBI also keeps its backlog order.

**Comment by westwoodad** [comment]
I agree, I think this would solve a lot of complaints that we get, people would much prefer to keep using the original ID in the new sprint.

Good idea!
